v5/core/container_authenticator_test.go

```diff
@@ -30,62 +29,64 @@ import (
 
 const (
 	// To enable debug logging during test execution, set this to "LevelDebug"
-	craTestLogLevel       LogLevel = LevelError
-	craMockCRTokenFile    string   = "../resources/cr-token.txt"
-	craMockIAMProfileName string   = "iam-user-123"
-	craMockIAMProfileID   string   = "iam-id-123"
-	craMockClientID       string   = "client-id-1"
-	craMockClientSecret   string   = "client-secret-1"
-	craTestCRToken1       string   = "cr-token-1"
-	craTestAccessToken1   string   = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VybmFtZSI6ImhlbGxvIiwicm9sZSI6InVzZXIiLCJwZXJtaXNzaW9ucyI6WyJhZG1pbmlzdHJhdG9yIiwiZGVwbG95bWVudF9hZG1pbiJdLCJzdWIiOiJoZWxsbyIsImlzcyI6IkpvaG4iLCJhdWQiOiJEU1giLCJ1aWQiOiI5OTkiLCJpYXQiOjE1NjAyNzcwNTEsImV4cCI6MTU2MDI4MTgxOSwianRpIjoiMDRkMjBiMjUtZWUyZC00MDBmLTg2MjMtOGNkODA3MGI1NDY4In0.cIodB4I6CCcX8vfIImz7Cytux3GpWyObt9Gkur5g1QI"
-	craTestAccessToken2   string   = "3yJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VybmFtZSI6ImhlbGxvIiwicm9sZSI6InVzZXIiLCJwZXJtaXNzaW9ucyI6WyJhZG1pbmlzdHJhdG9yIiwiZGVwbG95bWVudF9hZG1pbiJdLCJzdWIiOiJoZWxsbyIsImlzcyI6IkpvaG4iLCJhdWQiOiJEU1giLCJ1aWQiOiI5OTkiLCJpYXQiOjE1NjAyNzcwNTEsImV4cCI6MTU2MDI4MTgxOSwianRpIjoiMDRkMjBiMjUtZWUyZC00MDBmLTg2MjMtOGNkODA3MGI1NDY4In0.cIodB4I6CCcX8vfIImz7Cytux3GpWyObt9Gkur5g1QI"
-	craTestRefreshToken   string   = "Xj7Gle500MachEOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VybmFtZSI6ImhlbGxvIiwicm9sZSI6InVzZXIiLCJwZXJtaXNzaW9ucyI6WyJhZG1pbmlzdHJhdG9yIiwiZGVwbG95bWVudF9hZG1pbiJdLCJzdWIiOiJoZWxsbyIsImlzcyI6IkpvaG4iLCJhdWQiOiJEU1giLCJ1aWQiOiI5OTkiLCJpYXQiOjE1NjAyNzcwNTEsImV4cCI6MTU2MDI4MTgxOSwianRpIjoiMDRkMjBiMjUtZWUyZC00MDBmLTg2MjMtOGNkODA3MGI1NDY4In0.cIodB4I6CCcX8vfIImz7Cytux3GpWyObt9Gkur5g1QI"
+	containerAuthTestLogLevel       LogLevel = LevelError
+	containerAuthMockCRTokenFile    string   = "../resources/cr-token.txt"
+	containerAuthMockIAMProfileName string   = "iam-user-123"
+	containerAuthMockIAMProfileID   string   = "iam-id-123"
+	containerAuthMockClientID       string   = "client-id-1"
+	containerAuthMockClientSecret   string   = "client-secret-1"
+	containerAuthTestCRToken1       string   = "cr-token-1"
```

Comment:
One question: wouldn't be better to rename the `cr-token` to simply `token` or `container-token`?

Reply:
This is a good question. I know we've already messed with the naming plenty 🙂 but it's probably worth considering now that we've dropped the "compute resource" nomenclature from the user-facing code. This might apply to the `CRTokenFilename` property as well.

You do explain in the documentation that the token is a "compute resource token", so I think it's fine as is, but worth considering the simpler naming.

Reply:
I wanted to distinguish between the "cr token" and the "access token", hence the use of "cr token".   Even though we're now calling this the ContainerAuthenticator, the "token" is still considered to be a "compute resource token" (and that name can also be used in the VPCInstanceAuthenticator when we add that in as well).
If you guys are ok with it, I'd like to keep the current nomenclature ("cr token", "CRTokenFilename", etc.).

Reply:
That make sense so it's ok for me.

Reply:
That sounds good to me too 👍